src/VSCodeExtension/src/generator-qsharp/.eslintignore

```diff
@@ -0,0 +1,2 @@
+coverage
+**/templates
```

Comment:
With the move away from `generator-generator` to TypeScript in order to integrate better with the existing extension, I'd suggest dropping this file and adding an equivalent tslint configuration.

Reply:
Sounds good!